test/parallel/test-bootstrap-modules.js

```diff
@@ -95,12 +95,7 @@ const expectedModules = new Set([
   'NativeModule internal/process/pre_execution',
 ]);
 
-if (common.isMainThread) {
-  [
-    'NativeModule internal/idna',
-    'NativeModule url',
-  ].forEach(expectedModules.add.bind(expectedModules));
-} else {
+if (!common.isMainThread) {
```

Comment:
Can you revert please?

Reply:
Sure, but I don't really understand the reason of keeping it.

That is, should it be removed as semver-major change, or there is reason to keep `node:url` specifically?

Reply:
I should have said: "If you take my other suggestion, can you revert this, please?"

As I said in https://github.com/nodejs/node/pull/49590#pullrequestreview-1619359889, I think there is a reason to keep `node:url` specifically: we want it in the bootstrap path so it ends up in the snapsot.

Reply:
Yep, that's what I didn't understand: why we should keep `node:url` in snapshot? 😅 I.e. because it's breaking change in snapshot, or because we actually need it to be there?

Reply:
I'm not sure if it would be breaking, but it does introduce a noticeable change: not having `node:url` in the snapshot means userland application that use it will be (marginally) slower to start up, for a binary (marginally) smaller in size. It would merit a discussion on its own, and I'm not informed enough on the snapshot details to have a well-formed opinion; if you want to open a follow up PR proposing to remove it, we can ping the relevant folks and discuss the semverness there.

Reply:
Agreed; thanks for the clarification!